packages/portal/src/components/search/SearchQueryBuilderRule.vue

```diff
@@ -152,10 +152,15 @@
           ],
           modifier: [
             {
-              options: this.advancedSearchModifiers.map((mod) => ({
-                value: mod.name,
-                text: this.$t(`search.advanced.modifiers.${mod.name}`)
-              }))
+              options: this.advancedSearchModifiers.map((mod) => {
+                if (mod.name === 'exact' && (this.field !== this.fulltextFieldName)) {
```

Comment:
This should be derived instead of hard-coded. See comment re `supportsExact` in the mixin.